doc/source/cheat_sheet/.gitignore

```diff
@@ -0,0 +1 @@
+/.quarto/
```

Comment:
Is this needed?

Reply:
You will get "trash files" otherwise if you try to build it locally - this is were the build takes place (and it is kept, it is not deleted on completion AFAIK)